packages/build/src/config/config.ts

```diff
@@ -7,6 +8,18 @@ interface ManPageConfig {
   fileName: string;
 }
 
+// TODO: this is duplicated in update-notification-manager.ts
+export interface GreetingCTADetails {
+  chunks: {
+    text: string;
+    style?: string; // TODO: this is actually clr.ts/StyleDefinition
```

Comment:
Is addressing the open TODOs planned to be part of this PR?

Reply:
Good point - I'm not sure honestly. I can change the TODO to be merely a comment as I feel it's going to be significant undertaking to extract the common type and reference it in the cli-repl and the build projects. The actual permitted values are already enforced in json schema, so treating them as strings here is probably not the end of the world.